### PR TITLE
Update main figure 3

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Update main figure 3

In this PR, main figure 3 is updated to include the new model results. Small documentation changes were added to README related to the data used in this figure, to simplify and provide further clarity.

### CODE REVIEW

File: 2.evaluate_model/README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+We evaluate the final NF1 model on each split (train, validation, and test), each plate, and across all plates using the following metrics:
+
 - Precision
 - Recall
 - Accuracy
 - F1 score
 - Confusion matrices
-- Threshold precision and recall scores
+
+> **NOTE:** The precision and recall data cover the results from all the different parameter settings tested during the hyperparameter search. All other files contain the results from the final model (best hyperparameters).
```
Comment (MattsonCam) [inline 2.evaluate_model/README.md]:
```suggestion
> **NOTE:** The precision and recall thresholded data includes results from the stratified cross-validation test dataset, saved while searching for the optimal NF1 model hyper-parameter.  Data without the data split label `val` were not evaluated during hyper-parameter optimization.
```

Comment (MattsonCam) [reply 2.evaluate_model/README.md]: The `thresholded` keyword is used to distinguish precision and recall metrics from the pr curve data. Also, only one hyper-parameter was optimized. I hope that helps. Lmk if you have any questions.

File: 3.figures/figures/main_figure_3_model_eval.png
Comment (MattsonCam) [inline 3.figures/figures/main_figure_3_model_eval.png:1]: May also consider including the validation set in the other two figures

**Review by MattsonCam — APPROVED** [review]
LGTM @jenna-tomkinson, just some minor comments. Good job!

File: 2.evaluate_model/README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.evaluate_model/README.md]: 👍 added!

File: 3.figures/figures/main_figure_3_model_eval.png
Comment (jenna-tomkinson) [reply 3.figures/figures/main_figure_3_model_eval.png:1]: I think in a previous meeting that we agreed to just include validation for the PR curves, but that can be easily updated later if we agree we need to include that information.

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thanks for the review! I am merging now!
